bridetalk_project/src/components/atoms/fillinbox/fillinbox.css

```diff
@@ -0,0 +1,39 @@
+.form {
```

Comment:
This classname made me go in your project and search for what it is.
Couldn't it be `create-user-form` instead?